scout/server/blueprints/clinvar/templates/multistep_add_variant.html

```diff
@@ -184,7 +183,7 @@ <h3 class="fs-subtitle">The interpretation, or clinical significance, of the var
 
               <fieldset>
                 <h2 class="fs-title">Associated Conditions</h2>
-                <h3 class="fs-subtitle">The condition for which the variant is interpreted. Detailed information about reporting condition is available at <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/faq_submitters/#pheno" target="_blank" rel="noopener">https://www.ncbi.nlm.nih.gov/clinvar/docs/faq_submitters/#pheno</a>. If multiple conditions are submitted for a variant, this indicates that the variant was interpreted for the combination of conditions in the same individual(s).  i.e. this variant causes both condition A and condition B in the same individual. This scenario is most common for a new disease or syndrome that does not yet have a name and is described by several clinical features.  If you want to indicate that the variant has been interpreted for more than one condition, please submit these as separate records. i.e. this variant causes condition A in some individuals and causes disease B in other individuals. Provide only one name or identifier for a condition; do not provide multiple names or identifiers for the same condition."</h3>
+                <h3 class="fs-subtitle">The condition for which the variant is interpreted. Examples available at <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/spreadsheet/#condition" target="_blank" rel="noopener">ClinVar</a>"</h3>
```

Comment:
This is an easier explanation to the conditions formats

Reply:
👍